pyproject.toml

```diff
@@ -96,6 +95,10 @@ requires = ["setuptools>=42", "setuptools-scm>=7"]
 
 [tool.setuptools]
 packages = ["xarray"]
+include-package-data = false
+
+[tool.setuptools.package-data]
+"*" = ["*.py"]
```

Comment:
I _think_ we need something like this:

```
[tool.setuptools.packages.find]
include = ["xarray*"]
```

...since that change will preclude `*.pyi` files...

Reply:
ah yes! I had definitely misinterpreted the original issue